ts/components/wallet/PayWebViewModal.tsx

```diff
@@ -169,10 +179,21 @@ export const PayWebViewModal = (props: Props) => {
         contextualHelp={emptyContextualHelp}
         headerTitle={props.modalHeaderTitle}
       >
-        <ScreenContentHeader
-          title={I18n.t("wallet.challenge3ds.title")}
-          subtitle={I18n.t("wallet.challenge3ds.subtitle")}
-        />
+        <View
+          style={styles.descriptionContainer}
+          testID={"PayWebViewModal-description"}
+        >
+          <InfoBox
+            iconName={"io-info"}
+            iconColor={IOColors.bluegreyDark}
+            iconSize={24}
+          >
+            <Text style={styles.description}>
+              {I18n.t("wallet.challenge3ds.description")}
+            </Text>
```

Comment:
```suggestion
<Label weight={"Regular"} color={"bluegrey"}>
   {I18n.t("wallet.challenge3ds.description")}
</Label>
```
`Label` is a typography component that ensures certain IO styles and features
Could it possibile to add it in place of using custom `Text`?

Reply:
Thank you @Undermaken for the suggestion! I missed that component 🤦 and I believe it is the right choice here.

That said, `Label` component [has fixed 16px size](https://github.com/pagopa/io-app/blob/master/ts/components/core/typography/Label.tsx#L13) whereas from the designs we need 14px. 
Adding a screenshot here for you and @thisisjp to evaluate it:

![Screenshot 2021-06-25 at 09 30 55](https://user-images.githubusercontent.com/2094604/123387767-6d464880-d598-11eb-9811-e80bd5759795.png)

Reply:
Looks good to me, we'll review all the design library components later ;) thanks

Reply:
Updated https://github.com/pagopa/io-app/pull/3169/commits/82a268155eeb5e801ab9ceb93a1d6fc96be9e5f9